Makefile

```diff
@@ -20,11 +20,13 @@ osbs-build:
 	# hack/build.sh
 	./tmp/build/build.sh
 
-unit:
+unit: generate-mocks unit-test
```

Comment:
Does `unit` follow a convention? Maybe this command could be named `test`

Reply:
I don't think 'test' makes sense, since we want to leave room for more than one test type. In the future, we could also have something like 'functional' and 'component' that also refer to specific tests. At that point, we could use something like 'test' that would refer to 'unit' 'component' and 'functional' as a group.